libs/blocks/article-header/article-header.js

```diff
@@ -1,16 +1,15 @@
 import { createTag, getMetadata, getConfig } from '../../utils/utils.js';
-import { copyToClipboard } from '../../utils/tools.js';
```

Comment:
Is `copyToClipboard` in `tools.js` used outside of article header?

Reply:
No, not in milo but I didn't want to delete it in case a consumer is using it.